商品一覧機能の実装

mitoo0403 opened the pull request:
まだ商品出品されてない場合の画像
https://gyazo.com/f592e209cb94ef5ffc374e90fd8fb23e

商品出品した時の動画
https://gyazo.com/84d11fef3b26300c3f72a4f6c86a46be

Commit:
商品一覧機能の実装

Katsuno0814 commented on app/controllers/items_controller.rb:
```diff
@@ -3,7 +3,7 @@ class ItemsController < ApplicationController
 
   def index
-    @items = Item.includes(:user).order("created_at DESC")
+    @items = Item.all
```
> @items = Item.all

こちら、商品を全て表示するだけでなく、orderメソッドを用いて、出品した順に並び替えましょう！
前に記述された `@items = Item.includes(:user).order("created_at DESC")`が良いと思われます。

Katsuno0814 commented on app/views/items/index.html.erb:
```diff
@@ -174,6 +177,8 @@
         </div>
         <% end %>
       </li>
+     <% else %>
```
><% else %>

180行目のelseは不要になります。

Katsuno0814 reviewed (commented):
レビュー依頼ありがとうございます！
コメントしましたので、ご確認をお願いいたします！
また、プルリクエストにwhat,whyの記載がございません。
What（どのような実装をしているのか）と Why（なぜこの実装が必要なのか）を記載することで、第三者にブランチでの作業内容を伝えることが可能でございます。
記述してみましょう！

Commit:
修正①

コントローラー
Items.all→Item.includes(:user).order("created_at DESC")に変更
不必要なelseを削除
itemsテーブルにt.timestampsを追加

mitoo0403 commented:
どうしてこの機能が必要か？何故この機能が必要か？
この機能はトップページにログインしていなくても商品を見れる事で購買者に「どのような商品が出品されているか」を示す事ができる

NaturalRadio reviewed (commented):
レビュー依頼ありがとうございます！
LGTMとさせていただきます！実装お疲れ様でした！
引き続き実装頑張っていきましょう！
